core/src/main/java/jenkins/I18n.java

```diff
@@ -95,7 +95,7 @@ public HttpResponse doResourceBundle(StaplerRequest request) {
         String variant = request.getParameter("variant");
         
         try {
-            Locale locale = Locale.getDefault();
+            Locale locale = request.getLocale();
```

Comment:
http://docs.oracle.com/javaee/1.4/api/javax/servlet/ServletRequest.html#getLocale%28%29

Returns default server locale if one isn't provided by the client.